step-templates/iis-website-update-property.json

```diff
@@ -37,11 +41,11 @@
       }
     }
   ],
-  "LastModifiedOn": "2015-10-23T01:00:50.788+00:00",
-  "LastModifiedBy": "jmalczak",
   "$Meta": {
-    "ExportedAt": "2015-10-23T02:15:40.676Z",
-    "OctopusVersion": "2.6.5.1010",
+    "LastModifiedOn": "2016-05-30T11:42:50.429Z",
+    "LastModifiedBy": "ylashin78",

```

Comment:
The above two lines should be top-level properties of the JSON, not a part of the `$Meta` property.

Also, I just noticed that the `LastModifiedBy` property doesn't match your GitHub username `ylashin`.